langlib/lang.xml/src/main/ballerina/src/lang.xml/xml.bal

```diff
@@ -14,8 +14,17 @@
 // specific language governing permissions and limitations
 // under the License.
 
-const string XML_NAMESPACE_URI = "http://www.w3.org/XML/1998/namespace";
-const string XMLNS_NAMESPACE_URI = "http://www.w3.org/2000/xmlns/";
+# The namespace URI bound to the `xml` prefix.
+public const string XML_NAMESPACE_URI = "http://www.w3.org/XML/1998/namespace";
+# The namespace URI bound to the `xmlns` prefix.
+public const string XMLNS_NAMESPACE_URI = "http://www.w3.org/2000/xmlns/";
+
+# The expanded name of the `xml:space` attribute.
+public const string space = "{http://www.w3.org/XML/1998/namespace}space";
```

Comment:
Any particular reason why these public constants are in lower case?

Reply:
I guess these come from the spec.